samples/tutorial/Tutorial2.md

```diff
@@ -421,6 +427,61 @@ A workflow's props is similar to its state in a sense: any time the workflow is
 
 Note that unlike output, which is how a child sends events to its parent, props does not represent events. In fact, another way to think of props is another kind of state – the "public" part of its state, if you will.
 
+Inside `TodoListWorkflow` create a new `ListProps` class and update TodoListWorkflow to use these props. Also update the method signaure of `initialProps` and `render` :
```

Comment:
This step was missing. I had to cross reference completed code to see what to add.